app/db.ts

```diff
@@ -0,0 +1,374 @@
+import cli from "./cli.js";
+import mariadb from "mariadb";
+import { Run, StatusCodes } from "energy-label-types";
+import { exit } from "process";
+
+enum DBTYPES {
+	INT,
+	TEXT,
+	FK,
+	PK,
+}
+enum Tables {
+	PluginName = "PluginName",
+	Plugin = "Plugin",
+	BrowserName = "BrowserName",
+	Browser = "Browser",
```

Comment:
Er Browser: browser version? Eller hvordan er den anderledes end BrowserName?
```suggestion
	BrowserVersion = "BrowserVersion",
```

Reply:
Det er starten på browser dimention, derfor syntes jeg stadigvæk at den skal hede Browser